docs/gettingstarted.rst

```diff
@@ -170,17 +173,14 @@ Deploy on CERN infrastructure
 
       (reana) $ pip install reana-cluster
 
-9. Instantiate REANA cluster using CVMFS and CEPHFS:
-
-   Edit ``reana_cluster/backends/kubernetes/templates/persistentvolumeclaims/ceph.yaml``
-   and set ``spec.recources.requests.storage`` to the size you want the
-   CEPHFS shared volume to be. Note that it is possible to set CEPHS volume size via
-   cli by passing ``--cephfs-volume-size <size in GB>``. You can also set cluster URL
-   with option ``--url <REANA_cluster_URL>``.
+9. Instantiate REANA cluster using CEPHFS:
 
    .. code-block:: console
 
-      (reana) $ reana-cluster -f reana-cluster.yaml --cvmfs --cephfs --url reana.cern.ch init
+      (reana) $ reana-cluster -f reana-cluster.yaml \
+                              --cephfs
+                              --cephfs-volume-size <size in GB>
+                              --url reana.cern.ch init
```

Comment:
Does url need to be changed? If yes, I would make it ``<reana-instance-url.cern.ch>``.